tools-v2/pkg/cli/command/curvebs/check/server/server.go

```diff
@@ -0,0 +1,244 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: CurveCli
+ * Created Date: 2023-05-07
+ * Author: pengpengSir
+ */
+package server
+
+import (
+	"fmt"
+	"time"
+
+	cmderror "github.com/opencurve/curve/tools-v2/internal/error"
+	cobrautil "github.com/opencurve/curve/tools-v2/internal/utils"
+	basecmd "github.com/opencurve/curve/tools-v2/pkg/cli/command"
+
+	listchunkserver "github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/list/chunkserver"
+	copysetbs "github.com/opencurve/curve/tools-v2/pkg/cli/command/curvebs/status/copyset"
+	"github.com/opencurve/curve/tools-v2/pkg/config"
+	"github.com/opencurve/curve/tools-v2/pkg/output"
+	"github.com/opencurve/curve/tools-v2/proto/proto/copyset"
+	"github.com/opencurve/curve/tools-v2/proto/proto/topology"
+	"github.com/spf13/cobra"
+)
+
+const (
+	serverExample = `$ curve bs check server`
+)
+
+type ServerCommand struct {
+	basecmd.FinalCurveCmd
+	Rpc      *listchunkserver.ListChunkServerRpc
```

Comment:
Such an rpc can be independently used to make a command and then return the result so that other commands can reuse the rpc.